src/content/getting-started/capz-cluster-creation/index.md

```diff
@@ -0,0 +1,425 @@
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-clippy
+last_review_date: 2023-04-04
+---
+
+### Cluster types
+
+As of now, there are two kind of workload clusters which can be created.
+
+* A **public** cluster, which means that the Kubernetes API is directly exposed to the internet.
+* A **private** cluster, where the Kubernetes API is only exposed to an Azure internal network. 
+To access the Kubernetes API there, a connection must be either established to this internal Azure network or the network of the management cluster itself.
+
+### Prepare configuration
+
+In order to create a workload cluster in an installation using the cluster API Provider Azure you can follow below steps.
+
+All the relevant configuration items will be generated by using `kubectl gs template cluster`. For that a set of input configuration is needed.
+The configuration must be passed as file and will be validated against the [remote schema](https://github.com/giantswarm/cluster-azure/blob/main/helm/cluster-azure/values.schema.json) in [cluster-azure](https://github.com/giantswarm/cluster-azure). 
+
+#### public cluster
+
+To get a public cluster provisioned, below cluster configuration must be given:
+
+```yaml
+# filename: cluster.yaml
+metadata:
+  name: "jrp45"
+  description: "jrp45 test cluster"
+  organization: "multi-project"
+providerSpecific:
+  location: westeurope
+  subscriptionId: 6b1f6e4a-6d0e-4aa4-9a5a-fbaca65a4711
+```
+
+This will create a workload cluster in the subscription `6b1f6e4a-6d0e-4aa4-9a5a-fbaca65a4711` in Azure region `westeurope`.
+As these are the only two defined values for the cluster, all the defaults from the [remote schema](https://github.com/giantswarm/cluster-azure/blob/main/helm/cluster-azure/values.schema.json) will get applied.
+
+As all mandatory Apps a workload cluster needs, are bundled as `default-apps-azure`, a basic configuration must be also given.
+
+```yaml
+# filename: default-app.yaml
+clusterName: "jrp45"
+organization: "multi-project"
+```
+
+Given the above files, a configuration can be generated by running
+
+```nohighlight
+$ kubectl gs template cluster --provider=capz --cluster-config=cluster.yaml --default-app-config=default-app.yaml --output=generated-manifest.yaml
+```
+
+Above command will generate a configuration similar like
+
+```yaml
+# filename: generated-manifest.yaml
+---
+apiVersion: v1
+data:
+  values: |
+    metadata:
+      description: jrp45 cluster
+      name: jrp45
+      organization: multi-project
+    providerSpecific:
+      location: westeurope
+      subscriptionId: 6b1f6e4a-6d0e-4aa4-9a5a-fbaca65a4711
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: jrp45-user-values
+  namespace: org-multi-project
+---
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+  name: jrp45
+  namespace: org-multi-project
+spec:
+  catalog: cluster
+  config:
+    configMap:
+      name: ""
```

Comment:
is it normal that those values for the configmap are empty ? should it not reference the `above` configmaps ?

Reply:
there are "multiple" types of `configMaps` ( i don't understand all of them - tbh: i understand none of them :see_no_evil: ).

But in this case, the configMap with the cluster configuration is referenced in line `112-115`.

That's how the templating works (atm). it might be much more clearer if empty fields got removed but that's another story